DQMOffline/Trigger/plugins/TagAndProbeBtagTriggerMonitor.cc

```diff
@@ -0,0 +1,170 @@
+#include <math.h>
+
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "Geometry/Records/interface/TrackerTopologyRcd.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DQM/TrackingMonitor/interface/GetLumi.h"
+#include "CommonTools/TriggerUtils/interface/GenericTriggerEventFlag.h"
+
+#include "DataFormats/Common/interface/Handle.h"
+
+
+#include "DQMOffline/Trigger/plugins/TagAndProbeBtagTriggerMonitor.h"
+
+#include "TLorentzVector.h"
+
+
+// -----------------------------
+//  constructors and destructor
+// -----------------------------
+
+TagAndProbeBtagTriggerMonitor::TagAndProbeBtagTriggerMonitor( const edm::ParameterSet& iConfig )
+{
+  folderName_             = iConfig.getParameter<std::string>("dirname");
+  processname_            = iConfig.getParameter<std::string>("processname");
+  pathname_               = iConfig.getParameter<std::string>("pathname");
+  triggerobjbtag_         = iConfig.getParameter<std::string>("triggerobjbtag");
+  jetPtmin_               = iConfig.getParameter<double>("jetPtMin");
+  jetEtamax_              = iConfig.getParameter<double>("jetEtaMax");
+  tagBtagmin_             = iConfig.getParameter<double>("tagBtagMin");
+  probeBtagmin_           = iConfig.getParameter<double>("probeBtagMin");
+  triggerSummaryLabel_    = iConfig.getParameter<edm::InputTag>("triggerSummary");
+  triggerResultsLabel_    = iConfig.getParameter<edm::InputTag>("triggerResults");
+  triggerSummaryToken_    = consumes <trigger::TriggerEvent> (triggerSummaryLabel_);
+  triggerResultsToken_    = consumes <edm::TriggerResults>   (triggerResultsLabel_);
+  offlineCSVPFToken_      = consumes<reco::JetTagCollection> (iConfig.getParameter<edm::InputTag>("offlineCSVPF"));
+
+}
+
+TagAndProbeBtagTriggerMonitor::~TagAndProbeBtagTriggerMonitor()
+{
+
+}
+
+void TagAndProbeBtagTriggerMonitor::bookHistograms(DQMStore::IBooker     & ibooker,
+				 edm::Run const        & iRun,
+				 edm::EventSetup const & iSetup) 
+{  
+   
+  std::string currentFolder = folderName_ ;
+  ibooker.setCurrentFolder(currentFolder.c_str());
+  
+  pt_jet1_ = ibooker.book1D("pt_jet1","pt_jet1",60,0,600);
+  pt_jet2_ = ibooker.book1D("pt_jet2","pt_jet2",60,0,600);
+  
+  pt_probe_ = ibooker.book1D("pt_probe","pt_probe",60,0,600);
+  pt_probe_match_ = ibooker.book1D("pt_probe_match","pt_probe_match",60,0,600);
+  
+
+  discr_offline_btagcsv_jet1_ = ibooker.book1D("discr_offline_btagcsv_jet1","discr_offline_btagcsv_jet1",20,0,10);
```

Comment:
I would call it more generally btag (w/o CSV)
because in the upcoming future I'm expecting you will use (or other users will) different taggers ....